03.bowling/bowling.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def make_frames(shots)
+  frames = []
+  shots.each_slice(2) do |s|
```

Comment:
```suggestion
  frames = shots.each_slice(2).map do |s|
```

mapを使うとシンプルな実装にできそうです